Archives/VolFile.cpp

```diff
@@ -27,18 +27,18 @@ namespace Archives
 
 
 
-	const char* VolFile::GetInternalFileName(int index)
+	std::string VolFile::GetInternalFileName(int index)
```

Comment:
Yes, this makes a lot of sense. I can see how this changes ties in with other changes. The null termination issue for CLM internal names makes a `const char*` much more awkward to work with.